stores/peers.go

```diff
@@ -0,0 +1,220 @@
+package stores
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+	"time"
+
+	"go.sia.tech/coreutils/syncer"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+type (
+	dbSyncerPeer struct {
+		Model
+
+		Address      string `gorm:"unique;index:idx_syncer_peers_address;NOT NULL"`
+		FirstSeen    unixTimeMS
+		LastConnect  unixTimeMS
+		SyncedBlocks unsigned64
+		SyncDuration unsigned64
+	}
+
+	dbSyncerBan struct {
+		Model
+
+		NetCidr    string     `gorm:"unique;index:idx_syncer_bans_net_cidr;NOT NULL"`
+		Expiration unixTimeMS `gorm:"index:idx_syncer_bans_expiration;NOT NULL"`
+		Reason     string
+	}
+)
+
+var (
+	// TODO: use syncer.ErrPeerNotFound when added
```

Comment:
afaik it was added

Reply:
we have not tagged `coreutils` after v0.0.1 and upgrading to `@master` causes some changes I'd want to tackle in a separate PR that only upgrades `coreutils`